app/templates/agenda/agendaitems/agendaitem/index.hbs

```diff
@@ -69,4 +69,10 @@
     @allowEditing={{user-may "manage-agendaitems"}}
     @onSave={{this.saveGovernmentAreas}}
   />
+  <Mandatees::SecretaryPanel
```

Comment:
This is only inside the `if this.subcase` branch, for the first agendaitem ("Goedkeuring van het verslag") we won't show this. Is this desired? I think in theory they might still want to update the secretary.

Reply:
Not sure, @tdn what do you think?